capstone/src/main/java/com/google/sps/servlets/Student.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import java.util.List;
+
+/** Each instance of Student contains all necessary student information */
+public class Student {
+  private String name;
+  private int gradYear;
+  private String major;
+  private String email;
+  private List<String> clubs;
```

Comment:
nit: could you use ImmutableList instead? It's always preferable to make your class objects immutable, and then use getter/setter implementations if you really want to add something to the list

Reply:
Gotcha, just changed it back to ImmutableList